### PR TITLE
Threads info and UAF checker info

### PR DESCRIPTION
**Pull request by ghost**
Added UAF Check info to Readme.MD and Threads and LLVM Bitcode info.

### CODE REVIEW

File: community/Dockerfile [hunk 1]
```diff
@@ -21,7 +23,8 @@ RUN mkdir src
 WORKDIR src
 RUN git clone https://github.com/Boolector/btor2tools.git
 RUN git clone https://github.com/Boolector/boolector.git
-RUN git clone https://github.com/PLSysSec/sys.git
+RUN git clone https://github.com/Gavazzi1/sys
```
Comment (deian) [inline community/Dockerfile:26]: I'd prefer the official repo vs. fork

File: src/Static/Check.hs [hunk 2]
```diff
@@ -74,7 +75,9 @@ checkPaths config = do
 fork :: Checker a b () -> Checker a b ()
 fork act = do
   s0 <- get
-  void $ liftIO $ forkIO (curTGroup s0) $ evalChecker act s0
+  void $ liftIO $ do
+    setNumCapabilities 1 -- number of Threads
```
Comment (deian) [inline src/Static/Check.hs:79]: I don't think we want this. Definitely not fixed to 1, but more generally you can control this from the cli with flags to ghc, no?

File: README.md [hunk 3]
```diff
@@ -74,7 +74,17 @@ The tool takes several options:
...
 
+# Threading
+
+It seems Sys runs as many threads as cores you have. You can limit it here
+
+src/Static/Check.hs via setNumCapabilities N.
+
+# Getting Bitcode (LLVM)
+
+You can use wllvm or gllvm to get Bitcode and Assembly Language Format Files (*.ll)
```
Comment (deian) [inline README.md:87]: This is a nice add, I'd make it part of a separate PR. I have no idea what these are so links to them and how to use vs. say our `dclang` would be great.

**Review by deian — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
This PR is to document my tests to fix the OOM issue. Added some changes to process files in 2 files batch (I think it works, new to Haskell)

Want to see on which file, function it OOMs .... actually a cool feature would be to make a timeout in thread and when one file takes too long, it could be skipped.

This can be totally wrong, as my other ideas.

Just playing with it around.

**Comment by ghost** [comment]
Kind of solved with a script that restarts job after it was OOMed ... it knows where to start since processed files were renamed to (*.ll.done). Will see how it works ...

**Comment by deian** [comment]
@marcinguy it's awesome that you're playing with Sys! Keep it up! But please be mindful that we get emails and notifications about issues, PRs, and even every push and comment to a PR. So let's keep PRs to pull requests (where e.g. we review the code/docs to an issues) please.
